src/components/Editor/SearchBar.css

```diff
@@ -65,6 +65,22 @@
   border-radius: 3px;
 }
 
+.search-bottom-bar .search-modifiers button.close-btn.big img {
+  width: 12px;
+  height: 12px;
+}
+
+.search-bottom-bar .search-modifiers .pipe-divider {
+  height: 70%;
+  border: 1px solid #e0e0e1;
```

Comment:
We should probably use an existing variable color instead of explicitly setting them.